pom.xml

```diff
@@ -117,6 +122,11 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.9.10.3</version>
         </dependency>
+        <dependency>
```

Comment:
why are we depending on this still? if you are repackaging

Reply:
i'm repackaging (well, shading) using the maven shade plugin.  JAR comes out to be 1.7MB, which I think is still ok?

Reply:
if you're shading the jar, this is still declaring a dependency on it no? it should just be a dependency during the shading, not for the module itself

Reply:
discussed and resolved offline